src/index.js

```diff
@@ -0,0 +1,27 @@
+import { createServer } from 'http';
+import { config } from 'dotenv-flow';
+import { connectDB } from './services/db.js';
+import app from './app.js';
+import AppError from './errors/AppError.js';
+import logger from './services/logger.js';
+
+config({ path: 'src/config/' });
+connectDB()
+  .then(() => console.log('DB connected!'))
+  .catch(() => {
+    throw new AppError('Database connection error', 500, true);
```

Comment:
try not to hardcode the status codes and use the the constants from the errorManagment object